lib/provisioner/worker/plugins/providers/fog_provider/lib/fog_provider/google.rb

```diff
@@ -31,6 +31,9 @@ class FogProviderGoogle < Coopr::Plugin::Provider
   @server_confirm_timeout = 600
   @disk_confirm_timeout = 120
 
+  # Root disk size in GB
+  @root_disk_size = 10
```

Comment:
right, not yet... it would require figuring out some way to resize the partition/image without a reboot.